app/src/main/java/com/xinto/opencord/ui/widget/Message.kt

```diff
@@ -212,23 +230,27 @@ fun WidgetMessageContent(
     text: AnnotatedString,
     modifier: Modifier = Modifier,
 ) {
-    ProvideTextStyle(MaterialTheme.typography.bodyMedium) {
-        Text(
-            modifier = modifier,
-            text = text,
-            inlineContent = mapOf(
-                "emote" to InlineTextContent(
-                    placeholder = Placeholder(
-                        width = 20.sp,
-                        height = 20.sp,
-                        placeholderVerticalAlign = PlaceholderVerticalAlign.Center
-                    )
-                ) { emoteId ->
-                    OCAsyncImage(
-                        url = "${BuildConfig.URL_CDN}/emojis/$emoteId",
-                    )
-                }
+    Text(
+        modifier = modifier,
+        text = text,
+        inlineContent = textInlineContent()
+    )
+}
+
+@Composable
+private fun textInlineContent(): Map<String, InlineTextContent> {
+    val emoteSize = (LocalTextStyle.current.fontSize.value + 2f).sp
+    return mapOf(
+        "emote" to InlineTextContent(
+            placeholder = Placeholder(
+                width = emoteSize,
+                height = emoteSize,
+                placeholderVerticalAlign = PlaceholderVerticalAlign.Center
             )
-        )
-    }
+        ) { emoteId ->
+            OCAsyncImage(
+                url = "${BuildConfig.URL_CDN}/emojis/$emoteId",
+            )
+        }
```

Comment:
This image url should probably be coming from DiscordCdnService

Reply:
You're right, weird that I missed that

Reply:
nvm it's in #29 :trollface: